auto_usb_attach/__main__.py

```diff
@@ -16,26 +16,32 @@
 from .device import Device
 from .xenusb import XenUsb
 
+# Remove pylint warnings on Windows (just here for my development)
+if not callable(getattr(os, 'getuid')):
+    os.getuid = lambda: 0
+    os.setreuid = lambda x, y: False
+    os.geteuid = lambda: 0
+
```

Comment:
Delete this.

Reply:
Or, better yet, fail and exit in an OS that isn't running a xen hypervisor...  See #17